hack/kind-up.sh

```diff
@@ -53,6 +53,27 @@ parse_flags() {
   done
 }
 
+check_local_dns_records() {
+  local glgc_ip_address
+  glgc_ip_address=""
+
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    # Suppress exit code using "|| true"
+    glgc_ip_address=$(dscacheutil -q host -a name garden.local.gardener.cloud | grep "ip_address" | head -n 1| cut -d' ' -f2 || true)
+  elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    # Suppress exit code using "|| true"
+    glgc_ip_address="$(getent ahosts garden.local.gardener.cloud | cut -d' ' -f1 || true)"
+  else
+    echo "Warning: Unknown OS. Make sure garden.local.gardener.cloud resolves to 127.0.0.1"
+    return 0
+  fi
+    
+  if ! echo "$glgc_ip_address" | grep "127.0.0.1" ; then
```

Comment:
You can suppress the output here, otherwise the matched results will be printed on the stdout.

```suggestion
  if ! echo "$glgc_ip_address" | grep -q "127.0.0.1" ; then
```